lib/xcode.js

```diff
@@ -109,7 +109,8 @@ async function getVersionWithoutRetry () {
 
   // we want to read the CFBundleShortVersionString from Xcode's plist.
   // It should be in /[root]/XCode.app/Contents/
-  const plistPath = xcodePath.replace(XCODE_SUBDIR, "/Contents/Info.plist");
+  let plistPath = xcodePath.replace(XCODE_SUBDIR, "/Contents/Info.plist");
+  plistPath = plistPath.replace(new RegExp("/$"), "").trim();
```

Comment:
before I merge this I'd like to understand how `plistPath` could ever have a trailing slash at this point in the code. I thought that `getPath()` would always return a string without a trailing slash based on previous commits